src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReportDocHelper.java

```diff
@@ -258,9 +258,28 @@ private static StringBuilder getAdhocReportCompletedTypeRow(AdhocReportType adho
         return sb;
     }
 
+    private static StringBuilder getLiveCaseLoadReportSummaryHdr(ListingData listingData) {
+        var sb = new StringBuilder();
+        AdhocReportType localReportSummaryHdr = listingData.getLocalReportsSummaryHdr();
+        if (localReportSummaryHdr != null) {
+            sb.append("\"Multiples_Total\":\"").append(
+                    nullCheck(localReportSummaryHdr.getMultiplesTotal())).append(NEW_LINE);
+            sb.append("\"Singles_Total\":\"").append(
+                    nullCheck(localReportSummaryHdr.getSinglesTotal())).append(NEW_LINE);
+            sb.append("\"Total\":\"").append(
+                    nullCheck(localReportSummaryHdr.getTotal())).append(NEW_LINE);
+        }
+
+        return sb;
+    }
+
     private static StringBuilder getLiveCaseLoadReport(ListingData listingData) {
         var sb = new StringBuilder();
 
+        if (listingData.getLocalReportsSummaryHdr() != null) {
+           sb = getLiveCaseLoadReportSummaryHdr(listingData);
+        }
+
         if (listingData.getLocalReportsDetail() != null && !listingData.getLocalReportsDetail().isEmpty()) {
```

Comment:
This can be replaced with !CollectionUtils.isEmpty(listingData.getLocalReportsSummaryHdr()). It should check for both null and empty collection.